crates/sui-framework/tests/safe_tests.move

```diff
@@ -57,83 +61,87 @@ module sui::safe_tests {
     #[test]
     /// Ensure that all funds can be withdrawn by the owners
     fun test_safe_create_and_withdraw_funds_as_owner() {
-        let owner = @1337;
-        let scenario = ts::begin(@0x1);
+        let owner = TEST_OWNER_ADDR;
+        let scenario_val = ts::begin(TEST_SENDER_ADDR);
+        let scenario = &mut scenario_val;
 
         let initial_funds = 1000u64;
-        create_safe(&mut scenario, owner, initial_funds);
```

Comment:
I'm not particular about this choice to factor out the `&mut` for `scenario`, I just prefer it to be consistent with the choice I found in other tests (so the setup is mostly standard), e.g., https://github.com/MystenLabs/sui/blob/a6387e8ddf164c97b67718690fd082c433d8968d/crates/sui-framework/tests/coin_balance_tests.move#L50-L51.

Happy to update the other tests to do the `&mut scenario` instead.

Reply:
+1 for consistency, no strong opinions on which pattern we go for.